src/components/exercise/CreateExercise.js

```diff
@@ -15,14 +15,13 @@ const CreateExercise = () => {
         e.preventDefault()
         client
             .post(`/exercise`, { name, gitHubUrl, objectives: obj })
-
-        navigate('/posts')
+            .then (() => navigate('/exercise'))
     }
 
     const editObj = (e, i) => obj[i] = e
 
     return (
-        <div>
+        <div style={{ display: 'inline-block'}}>
```

Comment:
I won't block the PR for this but always favour creating a CSS class instead of doing styles this way